templates.go

```diff
@@ -28,6 +28,126 @@ import (
 )
 `))
 
+type jsHeaderParams struct {
+	ToolName      string
```

Comment:
Would be useful to add a brief `godoc` attribute description for these. I see they're present elsewhere, but this would take a small step towards documenting this tool a bit more directly.

Reply:
Added.